tests/pubsub_logging_test.py

```diff
@@ -41,6 +55,77 @@ def test_compat_urlsafe_b64encode(self):
         self.assertEqual(expected, result)
 
 
+class GetPubsubClientTest(unittest.TestCase):
+    """Tests for utils.get_pubsub_client function.
+
+    You have to set GOOGLE_APPLICATION_CREDENTIALS pointing to the
+    json file of the service account.
+    """
+    RETRY = 3
+
+    def test_get_pubsub_client_with_service_account(self):
+        """Tests the client obtained by the service account method."""
+        client = get_pubsub_client()
+        project = os.environ.get(TEST_PROJECT_ENV, DEFAULT_TEST_PROJECT)
+        client.projects().topics().list(project=project).execute(
+            num_retries=self.RETRY)
+        # Providing an Http object this time.
+        client = get_pubsub_client(http=httplib2.Http())
+        project = os.environ.get(TEST_PROJECT_ENV, DEFAULT_TEST_PROJECT)
+        client.projects().topics().list(project=project).execute(
```

Comment:
/nit probably you just want to confirm that client is not None here, since presumably these other tests are accomplished elsewhere.